### PR TITLE
Create CFH endpoint

### PR DESCRIPTION
**Pull request by jocontreras**
**PT Story:** https://www.pivotaltracker.com/story/show/143863917

**Changes proposed in this PR:**
Added model for CFH
Create controller for CFH

**What we have learned in this feature**
We learned to sanitized parameters and learned to do POST request test

### CODE REVIEW

**Review by fottander — APPROVED** [review]
Looks good!

File: app/controllers/api/v1/cfhs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    if @cfh.save
+      render json: { message: "successful"}, status: 204
+    end
+  end
+
+private
+  def cfh_params
+    params.permit(:title, :message, :status)
+  end
+end
```
Comment (AmberWilkie) [inline app/controllers/api/v1/cfhs_controller.rb]: You guys don't think you could call this something another programmer might possibly recognize?

**Review by AmberWilkie — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/cfhs_controller.rb (hunk 1 above)
Comment (AmberWilkie) [reply app/controllers/api/v1/cfhs_controller.rb]: Fucking like `Request` or something?

Comment (AmberWilkie) [reply app/controllers/api/v1/cfhs_controller.rb]: Or that word might be reserved. Maybe `HelpRequest`

**Comment by AmberWilkie** [comment]
Guys I meant to change that everywhere. You can't have a cfhs model - that's totally incomprehensible to someone that didn't literally write that code.

**Review by AmberWilkie — APPROVED** [review]

File: app/controllers/api/v1/help_requests_controller.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class Api::V1::HelpRequestsController < ApplicationController
+  def create
+    @help_request = HelpRequest.new help_request
+    if @help_request.save
+      render json: { message: "successful"}, status: 204
+    end
+  end
+
+private
+  def help_request
```
Comment (tochman) [inline app/controllers/api/v1/help_requests_controller.rb:10]: This should be called `help_request_params`

File: app/controllers/api/v1/help_requests_controller.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class Api::V1::HelpRequestsController < ApplicationController
+  def create
+    @help_request = HelpRequest.new help_request
+    if @help_request.save
+      render json: { message: "successful"}, status: 204
```
Comment (tochman) [inline app/controllers/api/v1/help_requests_controller.rb:5]: Why would you set the status to 204?

File: spec/requests/api/v1/helprequest_spec.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+      params = {
+        title: 'Burning',
+        message: 'Pants is burning',
+        status: true
+                }
+
+      post "/api/v1/help_requests", params
+
+      response.code.should eq "204"
```
Comment (tochman) [inline spec/requests/api/v1/helprequest_spec.rb:15]: Again, can't see any reason for the 204

File: spec/requests/api/v1/helprequest_spec.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  describe 'POST /v1/help_requests' do
+    it 'should collect call for help message from mobile client' do
+
+      params = {
+        title: 'Burning',
+        message: 'Pants is burning',
+        status: true
+                }
+
+      post "/api/v1/help_requests", params
```
Comment (tochman) [inline spec/requests/api/v1/helprequest_spec.rb:13]: THB, I can't see how this is working, I would write this as 
```ruby
post "/api/v1/help_requests", { params: params }
```
Didn't you get any errors or warnings when running your tests?

File: db/schema.rb [hunk 6]
```diff
@@ -10,11 +10,19 @@
...
-ActiveRecord::Schema.define(version: 20170418181155) do
+ActiveRecord::Schema.define(version: 20170419114516) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "help_requests", force: :cascade do |t|
+    t.string   "title"
+    t.string   "message"
+    t.boolean  "status"
```
Comment (tochman) [inline db/schema.rb:21]: Shouldn't this be `urgent`rather than `status`? `status` `true/false` does not make sense.

**Pull request merged**